<launch>
    <!-- パラメータ設定セクション nodeとは独立させて書く-->
    <param name="subscribe_topic_name" value="/velodyne_points"/>
    <param name="number_of_iterations" value="2"/>
    <param name="sample_rate" value="0.5"/>
    <param name="noise_variance" value="0.1"/>
    <param name="min_distance" value="2.5"/>
    <param name="max_distance" value="20"/>
    <param name="max_height" value="-1.3"/>
    <param name="spoofed_frame_name" value="output_pc"/>
    <param name="start_time" value="80"/>
    <param name="end_time" value="150"/>
    <param name="attackablity_save_dir" value="/home/rokuto/cbulid_ws/src/spoofing_sim/Attackablity"/>
    <node name="node1" pkg="matching_viewer" type="node1.py" output="screen" />
</launch>